source/utils.js

```diff
@@ -150,11 +157,7 @@ module.exports = {
     dirsToCheck.forEach(function getFixturePairs(dirToCheck) {
       var fixturesPath = path.join(context.testPath, dirToCheck);
 
-      try {
-        fs.readdirSync(fixturesPath).forEach(buildCases.bind(null, fixturesPath));
-      } catch (error) {
-        // throw new Error('Tester: Cannot read fixtures folder.' + error);
-      }
+      fs.readdirSync(fixturesPath).forEach(buildCases.bind(null, fixturesPath));
```

Comment:
Fixtures folder may not necessarily be there, i don't want to force having a fixtures folder next to a spec file. That's why there is a tryCatch here.